website/static/playground.js

```diff
@@ -96,8 +96,8 @@ window.onload = function() {
             prettierVersion;
         } else {
           link.href =
-            "https://github.com/j-f1/forked-prettier/blob/master/CHANGELOG.md#" +
-            prettierVersion;
+            "https://github.com/prettier/prettier/blob/master/CHANGELOG.md#" +
+            prettierVersion.replace(/\./g, '');
```

Comment:
![image](https://user-images.githubusercontent.com/8341033/34331330-c1ff3652-e960-11e7-8926-9b29a7340af7.png)

Reply:
we should change some default values :)